### ISSUE TITLE
Add support to CIFS version argument

### ISSUE THREAD

**Issue by guieiras** [dianlight/addon-plex#1]
## Problem/Motivation
I have a old NAS which only connects through classic CIFS/SMBv1 protocol and because of that I can't connect on add-on.
Allow configure SMB protocol version on add-on config

## Expected behavior
Connect successfully with SMBv1 servers using `vers=1.0` on mount arguments.

## Actual behavior
I can't set Samba protocol version using current config options

## Steps to reproduce
- Configure a v1.0 Samba Server
- Try to connect
- Check logs for `mount error(13): Permission denied`

## Proposed changes
- Add an option for cifs version on `plex/config.json`
```json
"options": {
  "cifsversion": "3.0"
},
"schema": {
  "cifsversion": "str"
}
```

- Get config on `plex/rootfs/etc/services.d/plex/run` and use it on `mount`
```bash
CIFS_VERSION=$(bashio::config 'cifsversion')
mount -t cifs -o username=$CIFS_USERNAME,password=$CIFS_PASSWORD,vers=$CIFS_VERSION $disk /$disk && \
```

**Comment by guieiras** [comment]
I have suggested to use version 3.0 on default options because it's the most recent. In my case, I will use `1.0` instead.

**Comment by kalxite** [comment]
i think i have the same issue, can you fix

**Comment by dianlight** [comment]
Fixed with PR #2
